test/BF16/matmul-bf16.mlir

```diff
@@ -1,5 +1,5 @@
-// RUN: tpp-opt %s -map-linalg-to-tpp -pre-bufferization -one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map"  -canonicalize -drop-equivalent-buffer-results -finalizing-bufferize -convert-linalg-to-tpp -convert-tpp-to-xsmm -convert-xsmm-to-func -convert-vector-to-scf -convert-scf-to-cf -convert-vector-to-llvm -convert-func-to-llvm -convert-memref-to-llvm -canonicalize -reconcile-unrealized-casts | \
-// RUN: mlir-cpu-runner \
+// RUN: tpp-opt %s -map-linalg-to-tpp -pre-bufferization -one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map"  -canonicalize -drop-equivalent-buffer-results -finalizing-bufferize -convert-linalg-to-tpp -convert-tpp-to-xsmm -convert-xsmm-to-func -convert-vector-to-scf -convert-scf-to-cf | \
```

Comment:
I generally don't understand what is the entry point of `tpp-run`. What is the reasoning of leaving this two conversions outside `-convert-vector-to-scf -convert-scf-to-cf`?

Reply:
I tried to keep only LLVM lowering passes in `tpp-run`. this can be moved around, but I don't want to assume what kind of IR we'll be creating with `tpp-opt`.

Reply:
Ok please document it.